app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  TEST_PASSWORD = "mentorspassword1234"
```

Comment:
I am confused by seeing this constant string in the class definition.  I am not seeing where this is used.